Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -383,6 +383,14 @@ def ap_find_benchmarks(bld, use=[]):
         return
 
     includes = [bld.srcnode.abspath() + '/benchmarks/']
+    to_remove = '-Werror=suggest-override'
```

Comment:
This seems more elaborate than it needs to be; why do you need `need_remove` at all?